bounce.py

```diff
@@ -312,7 +312,10 @@ def set_background(self, name):
     def pause(self):
         ''' Pause play when visibility changes '''
         if self._step_sid is not None:
-            GLib.source_remove(self._step_sid)
+            try:
+              GLib.source_remove(self._step_sid)
+            except Exception as e:
+              pass
```

Comment:
Why is there a try clause in an if statement?

Reply:
Something you added for development in your local environment shouldn't be part of your changes here.